### PR TITLE
UserMotion docs

### PR DESCRIPTION
**Pull request by namirali**
### Proposed changes
UserMotion Catalog Documentation

### Merge timing
ASAP

### CODE REVIEW

File: src/connections/destinations/catalog/actions-usermotion/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: UserMotion (Actions) Destination
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+[UserMotion](https://usermotion.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="\_blank”} is AI-powered predictive lead scoring software discovers intention signals and scores leads on potential to buy, expand, or churn.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-usermotion/index.md:7]:
```suggestion
[UserMotion](https://usermotion.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="\_blank”} is AI-powered predictive lead scoring software that discovers intention signals and scores leads on potential to buy, expand, or churn.
```

File: src/connections/destinations/catalog/actions-usermotion/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: UserMotion (Actions) Destination
+---
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/actions-usermotion/index.md:3]:
```suggestion
---
title: UserMotion (Actions) Destination
id: 6537b5da8f27fd20713a5ba8
beta: true
---
```

File: src/connections/destinations/catalog/actions-usermotion/index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+This destination is maintained by UserMotion. For any issues with the destination, [contact their Support team](mailto:support@usermotion.com).
+
+## Getting started
+
+1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="\_blank”} search for "UserMotion".
+2. Select UserMotion and click **Add Destination**.
+3. Select an existing Source to connect to UserMotion (Actions).
+4. Go to the [UserMotion dashboard](https://app.usermotion.com/?returnPath=/settings/integrations?provider=api){:target="\_blank"}, find and copy the **API key**.
+5. Enter the **API Key** in the UserMotion destination settings in Segment.
+6. Save your changes and enable the destination.
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/actions-usermotion/index.md:18]:
```suggestion
1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="\_blank”} search for "UserMotion (Actions)".
2. Select UserMotion (Actions) and click **Add Destination**.
3. Select an existing Source to connect to UserMotion (Actions).
4. Go to the [UserMotion dashboard](https://app.usermotion.com/?returnPath=/settings/integrations?provider=api){:target="\_blank"}, find and copy the **API key**.
5. Enter the **API Key** in the UserMotion destination settings in Segment.
6. Save your changes and enable the destination.
```

**Review by tcgilbert — CHANGES_REQUESTED** [review]
just a couple changes, but lgtm

**Review by namirali — COMMENTED** [review]
docs updated

**Review by forstisabella — APPROVED** [review]

**Review by tcgilbert — APPROVED** [review]

**Pull request merged**